libs/database/src/lib/factories/delayed-job.factory.ts

```diff
@@ -6,7 +6,9 @@ export const DelayedJobFactory = FactoryGirl.define(DelayedJob, async () => ({
   status: 'succeeded',
   statusCode: 200,
   payload: { "data": "test" },
-  total_content: 0,
-  processed_content: 0,
-  proccess_message: "test"
+  totalContent: 0,
+  processedContent: 0,
+  progressMessage: "test",
+  isCleared: false,
+  createdBy: '1290'
```

Comment:
As noted above, this should be an integer, not a string.

Reply:
Changed for all parts